### PR TITLE
improve(LM rewards): yUSD week 5 liquidity mining rewards

### PR DESCRIPTION
**Pull request by chrismaree**
**Overview**
This PR adds week 4's liquidity mining payouts. The script was run from from From  from Aug 24, 11:00:00 PM UTC @ block 10725993 to Aug 31, 11:00:00 PM UTC @ block 10771633.

**Details**
During this period there was a roll between two pools (addresses documented below). During the roll, the yUSD provided to both pools was considered. This roll occurred from the starting timestamp of the week until Aug 28 11:00:00 PM UTC @ block 10752010.

The roll period accounted for `(10752010 - 10725993) / (10771633 - 10725993) = 0.5700482033` of the total rewards to be paid for the week. Therefore there was 14251 UMA allocated to the roll duration and  10749 UMA allocated to the period after the roll.

**Re-creation**
You can re-generate this PRs output by running the following:
1. Generate the rolled payouts that consider both pools from Aug 24 to Aug 28. Use `14251` as the UMA per week. 
```
node ./scripts/liquidity-mining/CalculateRollBalancerLPRewards.js --fromBlock 10725993 --toBlock 10752010 --pool1Address="0x58EF3abAB72c6C365D4D0D8a70039752b9f32Bc9" --pool2Address="0xd2f574637898526fcddfb3d487cc73c957fa0268" --tokenName="yusdeth" --rollNum=1 --umaPerWeek=14251 --synth1Address="0x81ab848898b5ffD3354dbbEfb333D5D183eEDcB5" --synth2Address="0xB2FdD60AD80ca7bA89B9BAb3b5336c2601C020b4"
```

2. Generate the normal weekly payouts for the remaining period from Aug 28 until Aug 31. Use `10749` UMA per week.
```
node ./scripts/liquidity-mining/CalculateBalancerLPRewards.js --fromBlock 10752011 --toBlock 10771633 --poolAddress="0xd2f574637898526fcddfb3d487cc73c957fa0268" --tokenName="yusdeth" --week=5 --umaPerWeek=10749
```

3. Join the payout objects
```
node ./scripts/liquidity-mining/JoinRolledPayouts.js --week 5 --rollNum 1 --tokenName yusdeth
```

This will generate two files. the final payout is found in `packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/Week_5_Mining_Rewards.json` under `shareHolderPayout`

**Generated files**
The scripts creates two files in the output. 
`Week_5_Mining_Rewards.json` contains the final payout amounts under `shareHolderPayout`. This object contains the joined payouts from the rolled period (two pools) and the normal farming period (one pool). The `weeklyShareHolderPayoutBeforeRollJoin` value in `Week_5_Mining_Rewards.json` is the weekly amounts from the normal (non-roll) period and is kept for prosperity and to enable auditability. The `Roll_1_Mining_Rewards.json` file contains the output during the roll. The `shareHolderPayout` in this file is the payouts for each shareholder during the roll period. Again, this is kept for auditability.

### CODE REVIEW

File: packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/expiring-contract-rolls/Roll_1_Mining_Rewards.json [hunk 1]
```diff
@@ -0,0 +1,280 @@
+{
```
Comment (mrice32) [inline packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/expiring-contract-rolls/Roll_1_Mining_Rewards.json:1]: Sorry, can you explain what this file is relative to the other file?

File: packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/Week_5_Mining_Rewards.json [hunk 2]
```diff
@@ -0,0 +1,539 @@
...
+    "0xf56d610cbf3208ff6f009cb740beff4e9ef4d7ad": "1.122148503508754744",
+    "0xf59e58ff8b8ba7cbbb22e583b2d4be309e0adbd3": "2.242651556317490551",
+    "0xf627e5f4bad95a956468d8bb6ee20b119f992e96": "3.421881820741333588",
+    "0xf8ad2c3f153c1f9587643ba3698a284d32f9cf6e": "77.641219443771686895",
+    "0xfbb42577eb5e6c40af214bc457afe1922f55556b": "0.21552582381996938",
+    "0xfe62ad2613ed201459348b3e8cf9cc430be02c23": "5.30487417705070542",
+    "0xff3fc772434505abff38eecde3c689d4b0254528": "5.404054226582761962",
+    "0xadeeb9d09b8bcee10943198fb6f6a4229bab3675": "41.720449000633482295"
+  },
+  "weeklyShareHolderPayoutBeforeRollJoin": {
```
Comment (mrice32) [inline packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/Week_5_Mining_Rewards.json:343]: How does this section differ from the one above that's just called shareHolderPayout (is the first one a superset?)?

**Review by daywiss — APPROVED** [review]

File: packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/expiring-contract-rolls/Roll_1_Mining_Rewards.json (hunk 1 above)
Comment (chrismaree) [reply packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/expiring-contract-rolls/Roll_1_Mining_Rewards.json:1]: This file contains the roll output. the shareholder payout here is the payouts when the considering only the period over the two pools (the roll). This is why it's in the expiring contracts roll directory

File: packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/Week_5_Mining_Rewards.json (hunk 2 above)
Comment (chrismaree) [reply packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/Week_5_Mining_Rewards.json:343]: yup, this is a superset. this `weeklyShareHolderPayoutBeforeRollJoin` contains the output of the non-roll period before joining the roll period. it's kept for auditability.

Comment (chrismaree) [reply packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/Week_5_Mining_Rewards.json:343]: I updated the PR description to explain the different file outputs.

**Pull request merged**